ElasticCloudAdapter.py

```diff
@@ -0,0 +1,61 @@
+import os
+import paramiko
+import yaml
+
+
+# TODO: When paramiko is updated > 2.4.2 remove this warning squelch
```

Comment:
add a date to this, "last checked Mar 14 2019" or something

Reply:
Done.